File: tests/.hooks/csharp/custom-http/src/Speakeasy/CustomHttp/Hooks/HookRegistration.cs
namespace Speakeasy.CustomHttp.Hooks
{
    public static class HookRegistration
    {
        public static void InitHooks(IHooks hooks)
        {
            var customSecurityHook = new CustomSecurityHook();
            hooks.RegisterBeforeRequestHook(customSecurityHook);
        }
    }
}

File: tests/.hooks/php/secondary/src/Hooks/HookRegistration.php
<?php

declare(strict_types=1);

namespace alphabetically\early\Hooks;

class HookRegistration
{
    /**
     * @param  Hooks  $hooks
     */
    public static function initHooks(Hooks $hooks): void
    {
        $customClientHook = new CustomClientHook();
        $hooks->registerSDKInitHook($customClientHook);

        $customSecurityHook = new CustomSecurityHook();
        $hooks->registerBeforeRequestHook($customSecurityHook);
    }
}
